frontend/catalyst/utils/toml.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Module for abstracting which toml_load to use.
+"""
+
+import importlib.util
+
+# TODO:
+# Once Python version 3.11 is the oldest supported Python version, we can remove tomli
```

Comment:
```suggestion
# Once Python version 3.11 is the oldest supported Python version, we can remove tomlib
```

Reply:
This is actually tomlkit now: https://github.com/PennyLaneAI/catalyst/pull/369/commits/1733c99fffaec0dd80af3cedca9b89924f555ec8